mitodl/ocw-hugo-themes
# Course home page tweaks

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #29 

#### What's this PR do?
Adjusts some dimensions as described in the google doc linked to the issue

#### How should this be manually tested?
Go over each item in the google doc. View the course home page on desktop and confirm the change looks correct, and check on mobile to make sure nothing appears broken.

#### Screenshots (if appropriate)
![Screenshot from 2021-06-02 09-00-08](https://user-images.githubusercontent.com/863262/120484432-124a7880-c381-11eb-98fe-a15d7d78530c.png)
![Screenshot from 2021-06-02 09-00-57](https://user-images.githubusercontent.com/863262/120484439-14acd280-c381-11eb-8761-64ce7e25236f.png)
